src/components/Header.jsx

```diff
@@ -46,10 +46,7 @@ const Header = () => {
       )}
       {!isTabletOrMobile && (
```

Comment:
Suggest defining `paper` properties in a separate variable using a ternary to avoid code repetition.